### PR TITLE
Mobile version day4 request

### PR DESCRIPTION
**Pull request by AlphaNtihinduka**
The mobile version is composed of the use of a grid on the works section parent container and a flex box on individual cards. The branch composes of the works section, about myself section, and portfolio setup section respectively in the index.html file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,16 +15,171 @@
...
+                    <button type="button" class="see-prjct">See Project</button>
+                </div>
+               
+            </div>
+
+            <!-- CARD 4 (Multi-post stories)-->
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootCard4.png" alt="snapshoot">
+                    <h1 class="prjct-title multi-post">Multi-Post <br>Stories</h1>
```
Comment (JuliCarracedo) [inline index.html]: - [ ] Kindly change this `h1` tag to an `h2` tag. H1 tags are reserved to be used as a title to the website and should be used only once per page.

File: index.html [hunk 2]
```diff
@@ -15,16 +15,171 @@
...
     </header>
     <main>
+
+    <!-- THIS IS THE SECTION FOR MOBILE VERSION -->
+        <section class="grid-works">
+            <!-- CARD 1 (Tonic)-->
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootPortfolio.png" alt="snapshoot">
+                    <h1 class="prjct-title">TONIC</h1>
```
Comment (JuliCarracedo) [inline index.html]: - [ ] Kindly change this `h1` tag to an `h2` tag. H1 tags are reserved to be used as a title to the website and should be used only once per page.

File: index.html [hunk 3]
```diff
@@ -15,16 +15,171 @@
...
+                    <button type="button" class="see-prjct">See Project</button>
+                </div>
+               
+            </div>
+
+            <!-- CARD 3 (Tonic)-->
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootCard3.png" alt="snapshoot">
+                    <h1 class="prjct-title">TONIC</h1>
```
Comment (JuliCarracedo) [inline index.html]: - [ ] Kindly change this `h1` tag to an `h2` tag. H1 tags are reserved to be used as a title to the website and should be used only once per page.

File: index.html [hunk 4]
```diff
@@ -15,16 +15,171 @@
...
+                    <button type="button" class="see-prjct">See Project</button>
+                </div>
+               
+            </div>
+
+            <!-- CARD 2 (Multi-post stories)-->
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootCard2.png" alt="snapshoot">
+                    <h1 class="prjct-title multi-post">Multi-Post <br>Stories</h1>
```
Comment (JuliCarracedo) [inline index.html]: - [ ] Kindly change this `h1` tag to an `h2` tag. H1 tags are reserved to be used as a title to the website and should be used only once per page.

File: index.html [hunk 5]
```diff
@@ -15,16 +15,171 @@
...
+                </div>
+               
+            </div>
+        </section>
+
+
+        <!-- ABOUT SELF SECTION -->
+        <section class="about-section-container">
+            <div class="about-section-card">
+                <h1 class="name-intro">
```
Comment (JuliCarracedo) [inline index.html]: - [ ] Kindly change this `h1` tag to an `h2` tag. H1 tags are reserved to be used as a title to the website and should be used only once per page.

**Review by JuliCarracedo — CHANGES_REQUESTED** [review]
Hi @AlphaNtihinduka , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

[✔️](https://emojipedia.org/check-mark/) Professional PR
[✔️](https://emojipedia.org/check-mark/) Great job so far ap[plying the Figma design!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] you should update your README to look more professional. You're missing an explanation on how to install the project locally and the whole` .gitignore` section looks like something you pasted acccidentally.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -15,16 +15,171 @@
...
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootPortfolio.png" alt="snapshoot">
+                    <h2 class="prjct-title">TONIC</h2>
+                    <div class="prjct-info">
+                        <h6 class="canopy client-name">CANOPY</h6>
+                        <div class="dot-separator"></div>
+                        <h6 class="client-name">Back End Dev</h6>
+                        <div class="dot-separator"></div>
+                        <h6 class="client-name">2015</h6>
```
Comment (AmaduKamara) [inline index.html:52]: Your design is looking good 👍 so far, but there are a few changes to look into.

[ Required Changes ]

- [ ] Kindly move this section below your headline section  `<section class="hdln-section">`. The headline section is meant to be the first section in your portfolio that gives a brief description of you as a developer. Please fix that👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES  ♻️
Hi @AlphaNtihinduka, 

Good job so far!
You did great by making the changes the previous reviewer asked you to make. But there are some issues that you still need to work on to go to the next project but you are almost there!

ClappingLeonardoDicaprioGIF

## Highlights
✔️ No Linter errors
✔️ Design implemented nicely so far
✔️ Nice and descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- It would be nice you follow this [README Template](https://github.com/microverseinc/readme-template/blob/master/README.md) and place your personal contents in the sections provided in the template.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Kingstalux — COMMENTED** [review]
Hi @AlphaNtihinduka,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] (OPTIONAL) Kindly remove all unnecessary sections from your README like the sections for installing linters. This will make your README file more professional 👍 
- Your README

Annotation 2022-03-18 215324

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — APPROVED** [review]
Approved 👍

**Pull request merged**